### DISCUSSION TITLE
Welcome to stockplotr Discussions!

### DISCUSSION THREAD

**Discussion by Schiano-NOAA** [nmfs-ost/stockplotr#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iantaylor-NOAA** [comment]
Thanks for starting this discussion @Schiano-NOAA. I'm Ian Taylor, a stock assessment scientist at NWFSC and I'm hoping to use the satf package for the US West Coast Yellowtail Rockfish stock assessment that @okenk is leading. Our draft report is due on 27 April 2025.

A question I have is why there aren't more watchers of this repository? Is it because the folks who are likely to use the package are overwhelmed with messages and notifications, because until recently the messages have mostly been communication between the developers, because they don't know it exists, or something else?

**Reply by Schiano-NOAA** [reply]
@iantaylor-NOAA Thanks for joining! To answer your question, I think satf has not been promoted as much as asar due to it being a slightly different timeline. The package has made significant progress in the last couple months, so when it was presented publicly, the package was in very early stages. 

I do have a feeling this package is less known about, but hopefully through using asar, users will notice the tables and figures in the asar stock assessment report template are made using satf :)
